### PR TITLE
[SPARK-21286][Test] Modified StorageTabSuite unit test

### PR DESCRIPTION
**Pull request by Geek-He**
## What changes were proposed in this pull request?
The old unit test not effect

## How was this patch tested?
unit test

### CODE REVIEW

File: core/src/test/scala/org/apache/spark/ui/storage/StorageTabSuite.scala [hunk 1]
```diff
@@ -74,7 +74,7 @@ class StorageTabSuite extends SparkFunSuite with BeforeAndAfter {
     // Submitting RDDInfos with duplicate IDs does nothing
     val rddInfo0Cached = new RDDInfo(0, "freedom", 100, StorageLevel.MEMORY_ONLY, Seq(10))
     rddInfo0Cached.numCachedPartitions = 1
-    val stageInfo0Cached = new StageInfo(0, 0, "0", 100, Seq(rddInfo0), Seq.empty, "details")
+    val stageInfo0Cached = new StageInfo(0, 0, "0", 100, Seq(rddInfo0Cached), Seq.empty, "details")
```
Comment (cloud-fan) [inline core/src/test/scala/org/apache/spark/ui/storage/StorageTabSuite.scala]: why this test didn't fail before?

**Comment by cloud-fan** [comment]
ok to test

**Comment by HyukjinKwon** [comment]
Not a bid deal but I would like to suggest to fix the title to be more descriptive.

**Comment by HyukjinKwon** [comment]
Not a big deal but I would like to suggest to fix the title to be more descriptive.

**Review by srowen — APPROVED** [review]

**Comment by Geek-He** [comment]
What happened? why failed?

**Comment by jiangxb1987** [comment]
LGTM

**Comment by cloud-fan** [comment]
thanks, merging to master!

**Pull request closed without merging**